### PR TITLE
Added example game. Also added a license file that was missing from the ...

### PR DESCRIPTION
**Pull request by modeswitch**
...original.

### CODE REVIEW

File: examples/simple_canvas_game/js/game.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	update(delta / 1000);
+	render();
+
+	then = now;
+};
+
+// Let's play this game!
+reset();
+var then = Date.now();
+setInterval(main, 1); // Execute as fast as possible
```
Comment (dmose) [inline examples/simple_canvas_game/js/game.js]: requestAnimationFrame here?

File: examples/simple_canvas_game/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Simple Canvas Game</title>
+	</head>
+	<body>
+		<script src="js/game.js"></script>
```
Comment (dmose) [inline examples/simple_canvas_game/index.html:8]: Do we want to use require instead?  I don't think it'll make a difference to the volo optimization step, but we should check either way.

Comment (modeswitch) [inline examples/simple_canvas_game/index.html:8]: Require is overkill for this.

File: examples/simple_canvas_game/js/game.js (hunk 1 above)
Comment (modeswitch) [inline examples/simple_canvas_game/js/game.js]: Will fix and resubmit.

**Pull request merged**